proj_circle.yml

```diff
@@ -16,18 +20,9 @@ test:
     # imports check
     - isort **/*.py --check-only
     # pre-commit additional checks
-    - SKIP=prospector,isort pre-commit run --all-files
-    - >
-        DJANGO_SETTINGS_MODULE={{project_name}}.settings.local
-            python manage.py has_missing_migrations --ignore authtools;
-    - >
-        SECRET_KEY=$(python -c 'import uuid; print(uuid.uuid4().hex + uuid.uuid4().hex)')
-        DATABASE_URL='sqlite:///'
-        ALLOWED_HOSTS='.example.org'
-        SENDGRID_USERNAME='test'
-        SENDGRID_PASSWORD='test'
-        REDIS_URL='redis://'
-            python manage.py check --deploy --settings={{project_name}}.settings.production
+    - SKIP=prospector,isort,eslint,missing-migrations pre-commit run --all-files
+    - DJANGO_SETTINGS_MODULE={{project_name}}.settings.local_base python manage.py has_missing_migrations --ignore authtools;
+    - DJANGO_SETTINGS_MODULE={{project_name}}.settings.production SECRET_KEY=$(python -c 'import uuid; print(uuid.uuid4().hex + uuid.uuid4().hex)') DATABASE_URL='sqlite:///' ALLOWED_HOSTS='.example.org' SENDGRID_USERNAME='test' SENDGRID_PASSWORD='test' REDIS_URL='redis://' python manage.py check --deploy
```

Comment:
IMO this would be more readable split into multiple lines.

Reply:
It wasn't working multi-line... I'm not sure if it was an identation problem, but I couldn't make it work. I'll check this tomorrow!

Reply:
Multi-line support sucks a lot in YAML. Try `- >-` Instead of `- `:
```
- >-
  SECRET_KEY=$(python -c 'import uuid; print(uuid.uuid4().hex + uuid.uuid4().hex)')
  ...
```